packages/langchain-graph-retriever/pyproject.toml

```diff
@@ -108,7 +108,7 @@ cassandra = [
 ]
 chroma = [
     "langchain-chroma>=0.2.0",
-    "chromadb>=0.5.23",
```

Comment:
I think to get the depcheck to pass we'll want to either comment it to say chromadb is OK, or remove it -- IIUC, we're not importing it directly, and just checking to make sure it is installed (which I believe it should be as a dependency of langchain_chroma).